examples/http/nginx_status.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * this driver is based on nginx module "ngx_http_stub_status_module"
+ * You need to read this http://nginx.org/en/docs/http/ngx_http_stub_status_module.html 
+ * before using this driver
+ */
+// nginx status page config
+var nginxStatusHttpConfig = {
+    port: 10011,
```

Comment:
I thought it uses 8080 by default so please set the most likely/default value here

Reply:
This conversation is not resolved as you've not changed the NGINX port to a default one (80, 443, 8080, 8443)

Reply:
yes that's right, I changed it and I forgot to push